async_simple/coro/Lazy.h

```diff
@@ -43,7 +43,7 @@ class Lazy;
 // ```
 //
 // This would suspend the executing coruotine.
-struct Yield {};
+struct Yield_t {};
```

Comment:
Is this necessary? Why?

Reply:
Has reverted, it's related with asio.hpp header sequence.

Reply:
Can be removed.